CRM/Member/BAO/Membership.php

```diff
@@ -2357,22 +2357,16 @@ public static function mergeMemberships($mainContactID, $otherContactID, &$sqlQu
    *  - is_deceased
    *  - deceased_date
    *
-   * @param string $contactType
-   *
    * @return null|string
    *   $updateMembershipMsg string  status message for updated membership.
    */
-  public static function updateMembershipStatus($deceasedParams, $contactType) {
+  public static function updateMembershipStatus($deceasedParams) {
```

Comment:
There are no other uses of this function in universe, so safe to change signature to remove param no longer needed.